web/src/views/SubmissionPortal/Components/SubmissionList.vue

```diff
@@ -203,6 +219,31 @@ export default defineComponent({
               </v-icon>
             </v-btn>
           </template>
+          <template #[`item.menu`]="{ item }">
+            <v-menu>
+              <template #activator="{ on }">
+                <v-btn
+                  class="ml-1"
+                  icon="mdi-dots-vertical"
```

Comment:
I know you aren't done yet, but just wanted to point out that the `icon` prop is a boolean and doesn't take a string. It indicates a _type_ of button, in this case round and 'text' (transparent). Therefore using the icon prop makes the `text` prop redundant. 
Admittedly icon is not an intuitive name for this behavior.

Reply:
That makes a lot of sense why it was behaving unexpectedly. Thanks for pointing it out. I think that format is from vuetify 1 since I formatted it after those docs. I’m planning to go through and update all those kinds of things after looking at the vuetify2 docs some more.

Reply:
Made this update